src/main/java/edu/greenblitz/robotName/subsystems/arm/elbow/NeoElbow/NeoElbow.java

```diff
@@ -0,0 +1,76 @@
+package edu.greenblitz.robotName.subsystems.arm.elbow.NeoElbow;
+
+import com.ctre.phoenix6.signals.NeutralModeValue;
+import com.revrobotics.*;
+import edu.greenblitz.robotName.subsystems.Battery;
+import edu.greenblitz.robotName.subsystems.arm.elbow.ElbowConstants;
+import edu.greenblitz.robotName.subsystems.arm.elbow.ElbowInputsAutoLogged;
+import edu.greenblitz.robotName.subsystems.arm.elbow.IElbow;
+import edu.greenblitz.robotName.utils.motors.GBSparkMax;
+import edu.wpi.first.math.geometry.Rotation2d;
+
+import static edu.greenblitz.robotName.subsystems.arm.elbow.NeoElbow.NeoElbowConstants.SIMPLE_MOTOR_FEED_FORWARD;
+
+
+public class NeoElbow implements IElbow {
+
+    private GBSparkMax motor;
+
+    public NeoElbow(){
+        motor = new GBSparkMax(NeoElbowConstants.NEO_MOTOR_ID, CANSparkMaxLowLevel.MotorType.kBrushless);
+        motor.config(NeoElbowConstants.ELBOW_CONFIG_OBJECT);
+
+        motor.getPIDController().setFeedbackDevice(motor.getEncoder());
+        motor.enableSoftLimit(CANSparkMax.SoftLimitDirection.kForward, true);
+        motor.enableSoftLimit(CANSparkMax.SoftLimitDirection.kReverse, true);
+        motor.setSoftLimit(CANSparkMax.SoftLimitDirection.kReverse, ElbowConstants.BACKWARD_ANGLE_LIMIT.getRadians());
+        motor.setSoftLimit(CANSparkMax.SoftLimitDirection.kForward, ElbowConstants.FORWARD_ANGLE_LIMIT.getRadians());
+        motor.getReverseLimitSwitch(SparkMaxLimitSwitch.Type.kNormallyOpen).enableLimitSwitch(NeoElbowConstants.IS_REVERSE_LIMIT_SWITCH_ENABLE);
+        motor.getForwardLimitSwitch(SparkMaxLimitSwitch.Type.kNormallyOpen).enableLimitSwitch(NeoElbowConstants.IS_FORWARD_LIMIT_SWITCH_ENABLE);
+
+        resetAngle(Rotation2d.fromRotations(motor.getAbsoluteEncoder(SparkMaxAbsoluteEncoder.Type.kDutyCycle).getPosition()));
+    }
+
+    @Override
+    public void setPower(double power) {
+       motor.set(power);
+    }
+
+    @Override
+    public void setVoltage(double voltage) {
+        motor.setVoltage(voltage);
+    }
+
+    @Override
+    public void setIdleMode(NeutralModeValue idleMode) {
+        CANSparkMax.IdleMode neoIdleMode = (idleMode == NeutralModeValue.Brake ? CANSparkMax.IdleMode.kBrake : CANSparkMax.IdleMode.kCoast);
+        motor.setIdleMode(neoIdleMode);
+    }
+
+    @Override
+    public void resetAngle(Rotation2d position) {
+        motor.getEncoder().setPosition(position.getRadians());
+    }
+
+    @Override
+    public void moveToAngle(Rotation2d targetAngle) {
+        motor.getPIDController().setReference(targetAngle.getRadians(), CANSparkMax.ControlType.kPosition, NeoElbowConstants.PID_SLOT, SIMPLE_MOTOR_FEED_FORWARD.calculate(0));
+    }
+
+    @Override
+    public void standInPlace() {
+        setVoltage(SIMPLE_MOTOR_FEED_FORWARD.calculate(0));
```

Comment:
calculate(0)? I think not!

Reply:
why not? its a static feed forward because it stands in place, if you want to do PID then it reintroduces a bunch of problems with goal angles and moving in the periodic.